aws/resource_aws_default_route_table_test.go

```diff
@@ -53,33 +54,37 @@ func TestAccAWSDefaultRouteTable_basic(t *testing.T) {
 	})
 }
 
-func TestAccAWSDefaultRouteTable_disappears_Vpc(t *testing.T) {
-	var routeTable1 ec2.RouteTable
-	var vpc1 ec2.Vpc
+func TestAccAWSDefaultRouteTable_vpcDisappears(t *testing.T) {
```

Comment:
The more prevalent style for naming these is `_disappears_UPSTREAM`:

```console
$ ag --stats 'func TestAcc.*_disappears_' aws/*_test.go
...
29 matches
25 files contained matches
...
$ ag --stats 'func TestAcc.*_[^_]+Disappears' aws/*_test.go
...
9 matches
9 files contained matches
...
```

I will leave a note for myself to update the contributing guide with the naming recommendation.

```suggestion
func TestAccAWSDefaultRouteTable_disappears_Vpc(t *testing.T) {
```

Reply:
Relates:
* https://github.com/hashicorp/terraform-provider-aws/issues/16591
* https://github.com/hashicorp/terraform-provider-aws/pull/16597